build.gradle

```diff
@@ -29,12 +33,30 @@ buildscript {
     }
 }
 
-allprojects {
+allprojects { project ->
+
     repositories {
         maven { url 'https://dl.bintray.com/lisawray/maven'}
         google()
         jcenter()
     }
+
+    afterEvaluate {
+        if (project.hasProperty('android')) {
+            android {
+                // use a custom location for the Lint reports so that it's easy to archive them in Circle CI
+                lintOptions {
+                    htmlOutput file("$lintReportsDir/${project.name}/lint-report.html")
+                }
+                // it would be nice to do the same with test results/reports
+                // but it doesn't work: https://issuetracker.google.com/issues/37132023
+                testOptions {
```

Comment:
I left this here in the hope they will fix it soon but I can also get rid of it. What do you think?

Reply:
can leave it there, that's fine